packages/plugin-ext/src/main/browser/plugin-shared-style.ts

```diff
@@ -121,8 +122,9 @@ export class PluginSharedStyle {
     }
 
     static toExternalIconUrl(iconUrl: string): string {
-        if (iconUrl.startsWith('hostedPlugin/')) {
```

Comment:
we need to test for regressions in Electron, there should be a PR with `How to test` section before introducing it